.github/workflows/build.yml

```diff
@@ -215,12 +215,12 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - name: Install poetry
-        run: pipx install poetry
+        run: pipx install 'poetry<2.0'
 
       - name: Set up Python
         uses: actions/setup-python@v5
         with:
-          python-version: "3.9"
+          python-version: "3.11"
```

Comment:
If we bump the python version, can we do it everywhere in this file? Alternatively, we could choose not to bump.